src/vs/workbench/contrib/tasks/common/jsonSchema_v2.ts

```diff
@@ -45,6 +45,13 @@ const shellCommand: IJSONSchema = {
 	deprecationMessage: nls.localize('JsonSchema.tasks.isShellCommand.deprecated', 'The property isShellCommand is deprecated. Use the type property of the task and the shell property in the options instead. See also the 1.14 release notes.')
 };
 
+
+const hide: IJSONSchema = {
+	type: 'boolean',
+	description: nls.localize('JsonSchema.hide', 'Hide this task from the run task quickpick'),
```

Comment:
We use "quick pick" at https://github.com/microsoft/vscode/blob/0f3bbc794aa9c0c06a150097f4d61144c329b57d/src/vs/workbench/contrib/tasks/common/jsonSchema_v2.ts#L95